docs/sources/tempo/traceql/query-editor.md

```diff
@@ -11,20 +11,66 @@ keywords:
   - TraceQL
 ---
 
-# TraceQL query editor
+# Create TraceQL queries in Grafana
 
-You can use the TraceQL viewer and query editor in the Tempo data source to build queries and drill-down into result sets. The editor is available in Grafana’s Explore interface.
+You can use compose TraceQL queries in Grafana and Grafana Cloud using the Search in Explore and the TraceQL viewer and query editor in the Tempo data source.
+Both of these methods let you build queries and drill-down into result sets.
+
+To add TraceQL panels to your dashboard, refer to the [Traces panel documentation](/docs/grafana/latest/panels-visualizations/visualizations/traces/).
+
+To learn more about Grafana dashboards, refer to the [Use dashboards documentation](/docs/grafana/latest/dashboards/use-dashboards/).
 
 {{% admonition type="note" %}}
-To use the TraceQL query editor in Grafana, you need to enable the `traceqlEditor` feature flag. This feature is available starting in Grafana 9.3.2. The query editory is available automatically in Grafana Cloud.
+To use the TraceQL query editor in Grafana 9.3.2 and newer, you need to enable the `traceqlEditor` feature flag.
+
+To enable the Trace Search in Grafana 10 and newer, you need to enable the `traceqlSearch` feature flag.
+
+The query editor and search are available in Grafana Cloud without enabling a feature flag.
 {{% /admonition %}}
 
+## Create TraceQL queries using the query editor
```

Comment:
```suggestion
## Write TraceQL queries using the query editor
```

Additional question: what is the "Query Editor"? Is it the visual TraceQL builder of the "Query type" tab "Search"?

Reply:
This was a good point. We started off with only the TraceQL tab, so that was the TraceQL query editor. The Search tab was added on later, with the doc added to the same page. I've tried to clarify in this PR the differences. The query builder name is an excellent term to use to help clarify.